safe/utilities/direction_distance.py

```diff
@@ -0,0 +1,84 @@
+# coding=utf-8
+
+"""Direction and Distance Functionality."""
+
+import logging
+from qgis.core import QgsDistanceArea
+
+LOGGER = logging.getLogger('InaSAFE')
+
+
+def bearing_to_cardinal(angle):
+    """Get cardinality of an angle.
+
+    :param angle: Bearing angle
+    :return:
+    """
+    direction_list = [
```

Comment:
Perhaps it's also good to make the list translatable. You don't want to use N in Bahasa Indonesia. Although, I am not sure what's the translation will be.

Reply:
I think we only have 8 well-known directions, while in the list we have 18. Or maybe for Indonesia, we limit it to just 8 directions? I am not sure whether it is doable though